client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -60,7 +72,7 @@ const getBatchColumn = (
       width: 150,
       maxWidth: 150,
       maxLength: 50,
-      Cell: TextInputCell,
+      Cell: StocktakeTextInputCell,
```

Comment:
Did we need to create StocktakeTextInputCell ? Don't we just set isAutoFocus in column definition here ? (if batch column is moved, it won't be auto focused anymore, ¯\_(ツ)_/¯ I am not sure what the expected behaviour should be for this, still keep it on batch or always on the first editable column)

Reply:
I created the component because it's neater than trying to do it inline here..
Setting `isAutoFocus` to true for all batch input cells would mean the last row has focus and not the first, what you say about the column was helpful though! I've changed it so that this is a batch input cell only, and is only autoFocus on the first row